client/TracyProfiler.cpp

```diff
@@ -1069,7 +1069,8 @@ struct ProfilerThreadDataKey
         void* p = pthread_getspecific(m_key);
         if (!p)
         {
-            p = new ProfilerThreadData(GetProfilerData());
+            p = (ProfilerThreadData*)tracy_malloc( sizeof( ProfilerThreadData ) );
```

Comment:
Add `RPMallocInit init;` before this line.

Reply:
The `RPMallocInit init;` line could probably go in the `ProfilerThreadDataKey` constructor rather than the `get()` function. The constructor is called once immediately before the first ever call to `get()`

Reply:
I tried it both ways, and in the constructor, I still got the crash mentioned above.
But by putting it directly before `tracy_malloc`, the crash is gone!

Not sure the exact order-of-initialization stuff involved, but not sure I need to figure that out (:

Reply:
Yup, this would be low priority, as this initialization is not on the hot path.